frontend/src/components/editor/output/MarimoTracebackOutput.tsx

```diff
@@ -115,14 +117,11 @@ export const MarimoTracebackOutput = ({
             size="xs"
```

Comment:
we could add `{lastTracebackInfo && !isWasm() &&`, since i don't think pdb is well supported. i will need to do this on the traceback line-number feature too

Reply:
Good call. Done! Captured on the line num too